web/tests/integration/oidc/token.test.ts

```diff
@@ -419,4 +420,160 @@ describe("/api/v1/oidc/token", () => {
       })
     );
   });
+
+  test("Can get app_id using auth token", async () => {
+    const app_id = await testGetDefaultApp();
+    const { client_secret } = await setClientSecret(app_id);
+    const client_id = app_id;
+
+    // #region -> Can get app_id using auth token
+    await integrationDBExecuteQuery(
+      "INSERT INTO auth_code (app_id, auth_code, expires_at, nullifier_hash, scope, credential_type) VALUES ($1, $2, $3, $4, $5, $6)",
+      [
+        app_id,
+        "83a313c5939399ba017d2381",
+        "2030-09-01T00:00:00.000Z",
+        "0x000000000000000111111111111",
+        '["openid", "email"]',
+        "orb",
+      ]
+    );
+
+    const authToken = `Basic ${Buffer.from(
+      `${client_id}:${client_secret}`
+    ).toString("base64")}`;
+
+    const { req, res } = createMocks({
+      method: "POST",
+
+      headers: {
+        "Content-Type": "application/x-www-form-urlencoded",
+        authorization: authToken,
+      },
+
+      body: {
+        code: "83a313c5939399ba017d2381",
+        grant_type: "authorization_code",
+      },
+    });
+
+    await handleOIDCToken(req, res);
+    expect(res._getStatusCode()).toBe(200);
+    const { access_token } = res._getJSONData();
+
+    const { payload } = await jose.jwtVerify(
+      access_token,
+      await jose.importJWK(publicJwk, "RS256"),
+
+      {
+        issuer: process.env.JWT_ISSUER,
+      }
+    );
+
+    expect(payload).toEqual(
+      expect.objectContaining({
+        aud: app_id,
+      })
+    );
+    // #endregion
+
+    // #region -> Can get app_id using auth token without passing it directly
```

Comment:
this should be a different test case, and naming should be around using client_secret_post